2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_setup.py:_flush():69] setting env: {}
2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20211223_195129-1v42mwno/logs/debug.log
2021-12-23 19:51:29,174 INFO    MainThread:1973 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/tagger_text/wandb/run-20211223_195129-1v42mwno/logs/debug-internal.log
2021-12-23 19:51:29,175 INFO    MainThread:1973 [wandb_init.py:init():375] calling init triggers
2021-12-23 19:51:29,175 INFO    MainThread:1973 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/tags_text/tags_post_texts_v5.csv', 'checkpoint_path': '/root/konst/model_checkpoints/tagger_text', 'model_path': 'tag_txt_epoch_19_acc_0.5746_vs_0.5911_loss_1.2874_vs_1.7300.pth', 'batch_size': 128, 'epochs': 200, 'start_epoch': 20, 'test_size': 0.2, 'lam': 0, 'patience': 8, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2021-12-23 19:51:29,175 INFO    MainThread:1973 [wandb_init.py:init():424] starting backend
2021-12-23 19:51:29,175 INFO    MainThread:1973 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-23 19:51:29,187 INFO    MainThread:1973 [backend.py:ensure_launched():135] starting backend process...
2021-12-23 19:51:29,194 INFO    MainThread:1973 [backend.py:ensure_launched():140] started backend process with pid: 1998
2021-12-23 19:51:29,196 INFO    MainThread:1973 [wandb_init.py:init():429] backend started and connected
2021-12-23 19:51:29,199 INFO    MainThread:1973 [wandb_init.py:init():477] updated telemetry
2021-12-23 19:51:29,199 INFO    MainThread:1973 [wandb_init.py:init():500] communicating current version
2021-12-23 19:51:29,713 INFO    MainThread:1973 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-23 19:51:29,713 INFO    MainThread:1973 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-12-23 19:51:29,931 INFO    MainThread:1973 [wandb_init.py:init():540] starting run threads in backend
2021-12-23 19:51:31,865 INFO    MainThread:1973 [wandb_run.py:_console_start():1584] atexit reg
2021-12-23 19:51:31,865 INFO    MainThread:1973 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-12-23 19:51:31,867 INFO    MainThread:1973 [wandb_run.py:_redirect():1463] Redirecting console.
2021-12-23 19:51:31,868 INFO    MainThread:1973 [wandb_run.py:_redirect():1519] Redirects installed.
2021-12-23 19:51:31,868 INFO    MainThread:1973 [wandb_init.py:init():565] run started, returning control to user process
2021-12-24 00:43:28,105 INFO    MainThread:1973 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-12-24 00:43:28,106 INFO    MainThread:1973 [wandb_run.py:_restore():1526] restore
2021-12-24 00:43:30,693 INFO    MainThread:1973 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31977
  total_bytes: 31977
}

2021-12-24 00:43:30,929 INFO    MainThread:1973 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31977
  total_bytes: 31977
}

2021-12-24 00:43:31,067 INFO    MainThread:1973 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31977
  total_bytes: 55146
}

2021-12-24 00:43:31,168 INFO    MainThread:1973 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 4
}
pusher_stats {
  uploaded_bytes: 31977
  total_bytes: 58567
}

